bonus.md

```diff
@@ -0,0 +1,64 @@
+
+---
+
+### Bonus Round : Oceanic Navigation
+
+For SAUVC 2024, there is a bonus round with a single task called Oceanic Navigation. This was created to allow teams allow teams to experience the challenges of operating in an oceanic environment. The bonus round will be held in the [TCOMS Ocean Basin](https://www.tcoms.sg/), a cutting edge facility for testing and developing marine technology. The basin is located at the [National University of Singapore](https://www.nus.edu.sg/).
+
+![arena](img/ocean-basin.jpg)
```

Comment:
Best not to use any TCOMS image, lets just keep a blank image here as placeholder until we get approval from TCOMS to use an image.